test/recipes/30-test_evp_data/evpciph_aes_common.txt

```diff
@@ -1259,6 +1259,19 @@ IV = 9a785634120000000000000000000000
 Plaintext = 000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f404142434445464748494a4b4c4d4e4f5051
 Ciphertext = edbf9dace45d6f6a7306e64be5dd824b2538f5724fcf24249ac111ab45ad39233ad6183c66fa548a3cdf3e36d2b21ccdc6bc657cb3aeb87ba2c5f58ffafacd765ecc4c85c0a01bf317b823fbd6111956d0a0
 
+# To cover the branches of assembly code of aes_v8_xts_encrypt(decrypt)
```

Comment:
Where does the data come from? are they from specs? or are generated by oneself?

Reply:
It's come from [line 1029](https://github.com/openssl/openssl/pull/20797/files/eeaf34baac3a42ca28eaddea0b7d983163788b17#diff-556bad09eca86c009d60c31a75480597ed6d89aa40416c97dc550d968571cd7bR1029). It's just the same but with half length of input/output to cover the branches of the assembly code.